src/snapshots/001.txt

```diff
@@ -0,0 +1,23 @@
+~~~META
+description=Example to develop the snapshot methodology, includes FORMATTED section
+~~~SOURCE
+module     [
+# some crazy formatting
+ foo,
+     ]
+
+foo =
+
+    "one"
+~~~FORMATTED
+module [foo]
```

Comment:
Not that it matters for submitting this, but I think you already found a formatting bug. The comment disappeared.

Reply:
Yeah.. I noticed a few of these while messing with it. There's also plenty of fun ways to blow the parser up. It's nice and easy with this tool to find them.

Reply:
They are going to disappear for now.  It's not really a bug - just functionality that is missing :-)